cobra/flux_analysis/sampling.py

```diff
@@ -496,8 +592,7 @@ def __getstate__(self):
         return d
 
 
-def sample(model, n, method="optgp", processes=1, seed=None,
-           solver=None, **solver_kwargs):
+def sample(model, n, method="optgp", processes=1, seed=None):
```

Comment:
for larger models, sampling obviously takes quite a while, I think a progress bar would help.. tqdm is such a nice package perhaps it can be allowed to be a hard requirement even, what do you think? Can also be left for a different pr of course.